src/taco-cli/test/help.ts

```diff
@@ -59,14 +58,14 @@ describe("help for a command", function (): void {
     }
 
     before(() => {
-        previous = process.env["TACO_UNIT_TEST"];
         process.env["TACO_UNIT_TEST"] = true;
     });
 
     after(() => {
         // We just need to reset the stdout just once, after all the tests have finished
         process.stdout.write = stdoutWrite;
-        process.env["TACO_UNIT_TEST"] = previous;
+        // Reset mocked out resources
+        process.env["TACO_UNIT_TEST"] = false;
     });
```

Comment:
Does this change mean that we had tests not cleaning after running? Why do we force the "false" value and not the previous one anymore?